GitUpdateScript.sh

```diff
@@ -13,7 +13,7 @@ SECONDS=0  #set seconds counter back to 0
 function GoToGitRepositoryDirectory(){
 
 	cd $home_dir${file_dir:1:-4} #change to git dir
-	pwd #pint dir to screen
+	echo "Starting git repository refresh @" $(pwd) #print dir to screen
 }
 function FindsWhatBranchRepositoryIsOn() {
 	if [ "$branch_option" = "* $branch_I_want" ] #note "* " is used to show branch selection
```

Comment:
I'd recommend using `[[` over `[`. It's not POSIX compliant, but works in `bash`, `ksh`, and `zsh`, and is more feature-rich.

[For your reading](https://unix.stackexchange.com/a/32211/67145).

Reply:
Thanks will do